lib/lotus/commands/db.rb

```diff
@@ -0,0 +1,69 @@
+require 'lotus/router'
+require 'lotus/model'
+
+module Lotus
+  module Commands
+    class DB
+
+      DEFAULT_STEP = 1 
+      APP_MIGRATION_DIRECOTRY = "apps/web/db/migrations".freeze
+      MIGRATION_DIRECTORY = "db/migrations".freeze
+
+      def initialize(name=nil, environment)
+        @name         = name
+        @environment  = environment
+        @options      = @environment.to_options
+        load_environment
+        @migrator     = Lotus::Model::Migrator.new(adapter_config)
+      end
+
+      def migrate
+        @migrator.migrate(directory: migration_directory)
+      end
+
+      def rollback
+        @migrator.rollback(directory: migration_directory, step: step)
+      end
+
+      private 
+
+      def config
+        if @name
+          app_constant = Lotus::Utils::Class.load_from_pattern!(Lotus::Utils::String.new(@name).classify)
+          Lotus::Utils::Class.load_from_pattern!("#{app_constant}::Application").load!
+          Lotus::Utils::Class.load_from_pattern!("#{app_constant}::Model").configuration
+        else
+          Lotus::Model.configuration
+        end
+      end
+
+      def migration_directory
+        if @name 
+          pwd.join(APP_MIGRATION_DIRECOTRY)
```

Comment:
we can't hardcode directory. A lotus app can have many apps, the `web` is just a default app that is created, what if user want to run it against other apps? For the first phase of this feature, can you please remove the support for running migration against apps? I want to get migration done for the lib/app folder done correctly. Then we could discuss how we go about implementing the migration for apps, I am thinking of syntax `bundle exec lotus db migrate --app=web`